socketdock/api.py

```diff
@@ -42,8 +42,9 @@ async def status_handler(request: Request):
 @api.post("/socket/<connectionid>/send")
 async def socket_send(request: Request, connectionid: str):
     """Send a message to a connected socket."""
-    LOGGER.info("Inbound message for %s", connectionid)
-    LOGGER.info("Existing connections: %s", active_connections.keys())
+    if LOGGER.isEnabledFor(logging.INFO):
+        LOGGER.info("Inbound message for %s", connectionid)
+        LOGGER.info("Existing connections: %s", active_connections.keys())
```

Comment:
The `if` should not be required here; this is how `LOGGER.info` functions. If the log level is below `INFO`, the log will not be emitted and the parameters will not be stringified.